cowprotocol/cowswap
# chore(advanced-orders): read sell amount from url for TWAP

# Summary

While SWAP and LIMIT orders can take `sellAmount` (and `buyAmount`) from the url params, ADVANCED did not.

This PR is a very quick and dirty solution to make it available

Next step is to refactor all 3 into a single location as it's pretty much copy and paste.
For now I'm implementing it this way to not delay it any further.

# To Test

1. Load the app on advanced orders with the url param `sellAmount=2` or whatever amount (example: https://swap-dev-git-sell-amount-url-for-twap-cowswap.vercel.app/#/1/advanced-orders/WETH?sellAmount=4)
* The amount set in the url should be set onto the sell amount
* The parameter should be removed from the URL
